### ISSUE TITLE
Crash on play videos

### ISSUE THREAD

**Issue by fddaineze** [NGoedix/VideoPlayer#19]
[crash-2023-06-20_17.20.41-client.txt](https://github.com/NGoedix/WatchVideoMod/files/11806260/crash-2023-06-20_17.20.41-client.txt)

[latest.log](https://github.com/NGoedix/WatchVideoMod/files/11806316/latest.log)

the crash happens randomly, I had 2 instances of mine open during this test, one played the video and froze at the end of it, while the other crashed as soon as the command was executed.

For the open, the freeze lasted and I had to force the end of the game.

follow attached log.

```
Description: Unexpected error
java.lang.NoClassDefFoundError: me/lib720/caprica/vlcj4/player/base/MediaPlayerEventListener
...
```

**Comment by NGoedix** [comment]
Complete logs please

**Comment by fddaineze** [comment]
> Complete logs please

attached.

**Comment by NGoedix** [comment]
That's the crash, not the latest.log

**Comment by fddaineze** [comment]
> That's the crash, not the latest.log

Sorry
[latest.log](https://github.com/NGoedix/WatchVideoMod/files/11806308/latest.log)

**Comment by NGoedix** [comment]
You are using VideoPlayer 2.0.1 but today i released the version 2.0.3

https://www.curseforge.com/minecraft/mc-mods/video-player/files/4599346
